modules/core/src/main/java/org/apache/ignite/internal/util/ExceptionUtils.java

```diff
@@ -951,7 +951,7 @@ <T extends Throwable> T copy(MethodHandle constructor, UUID traceId, int code, S
      * </code></pre>
      *
      */
-    private static class UtilException extends Throwable {
+    private static class UtilException extends RuntimeException {
```

Comment:
Could you please clarify this change? What is the reason for this? Just in case.

Reply:
>  Currently this exception is not the one you would usually catch but it seems it's designed exactly to be caught.
This exception was introduced due to internal technical needs. It is not assumed that it can be thrown directly.
However, in this particular case, the exact type does not matter, we can use `RuntimeException`.